example/test/movie_response_test.dart

```diff
@@ -1,90 +1,90 @@
+//       _fakeMovieFetcher.fetchMovies().listen(expectAsync1((event) => expect(
+//             event,
+//             MoviesResponse.success(movies: movies),
+//           )));
+//     });
 
-    test('fetch movies should emit MovieResponse.unauthorized', () async {
-      when(_mockClient.get(
-        '$_baseUrl/popular?api_key=fakeApiKey',
-      )).thenAnswer(
-        (_) async => http.Response(await _getMovieJson(), 401, headers: {
-          HttpHeaders.contentTypeHeader: 'application/json; charset=utf-8'
-        }),
-      );
+//     test('fetch movies should emit MovieResponse.unauthorized', () async {
+//       when(_mockClient.get(
+//         '$_baseUrl/popular?api_key=fakeApiKey',
+//       )).thenAnswer(
+//         (_) async => http.Response(await _getMovieJson(), 401, headers: {
+//           HttpHeaders.contentTypeHeader: 'application/json; charset=utf-8'
+//         }),
+//       );
 
-      _fakeMovieFetcher.fetchMovies().listen(expectAsync1((event) => expect(
-            event,
-            MoviesResponse.unauthorized(),
-          )));
-    });
+//       _fakeMovieFetcher.fetchMovies().listen(expectAsync1((event) => expect(
+//             event,
+//             MoviesResponse.unauthorized(),
+//           )));
+//     });
 
-    test('fetch movies should emit MovieResponse.noNetwork', () async {
-      when(_mockClient.get(
-        '$_baseUrl/popular?api_key=fakeApiKey',
-      )).thenThrow(SocketException(''));
+//     test('fetch movies should emit MovieResponse.noNetwork', () async {
+//       when(_mockClient.get(
+//         '$_baseUrl/popular?api_key=fakeApiKey',
+//       )).thenThrow(SocketException(''));
 
-      _fakeMovieFetcher.fetchMovies().listen(expectAsync1((event) => expect(
-            event,
-            MoviesResponse.noNetwork(),
-          )));
-    });
+//       _fakeMovieFetcher.fetchMovies().listen(expectAsync1((event) => expect(
+//             event,
+//             MoviesResponse.noNetwork(),
+//           )));
+//     });
 
-    test('fetch movies should emit MovieResponse.unexpectedException',
-        () async {
-      final exception = Exception('Unexpected Error Occured');
+//     test('fetch movies should emit MovieResponse.unexpectedException',
+//         () async {
+//       final exception = Exception('Unexpected Error Occured');
 
-      when(_mockClient.get(
-        '$_baseUrl/popular?api_key=fakeApiKey',
-      )).thenThrow(exception);
+//       when(_mockClient.get(
+//         '$_baseUrl/popular?api_key=fakeApiKey',
+//       )).thenThrow(exception);
 
-      _fakeMovieFetcher.fetchMovies().listen(expectAsync1((event) => expect(
-            event,
-            MoviesResponse.unexpectedException(exception: exception),
-          )));
-    });
-  });
-}
+//       _fakeMovieFetcher.fetchMovies().listen(expectAsync1((event) => expect(
+//             event,
+//             MoviesResponse.unexpectedException(exception: exception),
+//           )));
+//     });
+//   });
+// }
```

Comment:
Uncomment these tests and try running them.

Reply:
Fixed